ptyprocess/ptyprocess.py

```diff
@@ -190,6 +190,10 @@ def spawn(
 
         Dimensions of the psuedoterminal used for the subprocess can be
         specified as a tuple (rows, cols), or the default (24, 80) will be used.
+
+        By default, all file descriptors except 0, 1 and 2 are closed. This
+        behavior can be overridden with pass_fds, a list of file descriptors to
```

Comment:
Nitpick: you document it as a list, but it will only work with a tuple, because you concatenate it to a tuple (`pass_fds + (exec_err_pipe_write,)`). Let's cast it to a set, like subprocess does; that should work with any sequence, and also eliminates any odd behaviour with duplicate entries.

Reply:
Right!